data/en/getlocaleinfo.json

```diff
@@ -4,29 +4,34 @@
 	"syntax":"getLocaleInfo()",
 	"returns":"struct",
 	"related":["getLocaleDisplayName","getLocale","getLocaleCountry","getLocaleLanguage"],
-	"description":"A short description that describes what the tag or function does.",
-	"discouraged":"Only add this key if this tag/function is discouraged by the community.",
+	"description":"Returns a structure containing info to a specific locale.\r\n\r\nThis function combines the locale functions getLocaleCountry, getLocaleDisplayName, and getLocaleLanguage to a single function.",
 	"params": [
-		{"name":"locale","description":"Geographic/language locale value","required":true,"default":"getLocale()","type":"string","values":["en","de_DE","de_ch","..."]},
-		{"name":"displayLocale","description":"Locale's display name/Output language","required":false,"default":"getLocaleDisplayName()","type":"string","values":["English","German","French","..."]}
+		{"name":"locale", "description": "Geographic/language locale value, where the format is a combination of an <a href=\"https://en.wikipedia.org/wiki/ISO_639-1\">ISO 639-1</a> code and an optional <a href=\"https://en.wikipedia.org/wiki/ISO_3166-1\">ISO 3166-1</a> code separated by a dash or an underscore.", "required": false, "default": "getLocale()", "type":"string", "values": ["en","de-DE","de-CH","..."]},
```

Comment:
Let's get rid of the HTML in here as well.

Reply:
I'm not sure: Does it parse markdown in there? I guess, not.

So in which way can we link that or should we do an i18n guide?